bot/utils/time.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+
+from dateutil.relativedelta import relativedelta
+
+
+# All these functions are from https://github.com/python-discord/bot/blob/master/bot/utils/time.py
+def _stringify_time_unit(value: int, unit: str) -> str:
+    """
+    Returns a string to represent a value and time unit, ensuring that it uses the right plural form of the unit.
+
+    >>> _stringify_time_unit(1, "seconds")
+    "1 second"
+    >>> _stringify_time_unit(24, "hours")
+    "24 hours"
+    >>> _stringify_time_unit(0, "minutes")
+    "less than a minute"
+    """
+    if unit == "seconds" and value == 0:
+        return "0 seconds"
+    elif value == 1:
+        return f"{value} {unit[:-1]}"
+    elif value == 0:
+        return f"less than a {unit[:-1]}"
+    else:
+        return f"{value} {unit}"
+
+
+def humanize_delta(delta: relativedelta, precision: str = "seconds", max_units: int = 6) -> str:
```

Comment:
Arrow library has `humanize()` function which can do this.

Reply:
I haven't looked into the differences so this isn't my opinion on it, but I noticed this touches a previously reviewed aspect, which you can find kosa's response to here:

> Our version works a lot better, the `arrow` version is clunky and not customizable.

https://github.com/python-discord/sir-lancebot/pull/462#discussion_r588906493

Reply:
Alright! Didn't see the earlier reviews.